Library/Formula/gpg-agent.rb

```diff
@@ -33,9 +33,46 @@ def install
     system "make", "install"
   end
 
+  def caveats; <<-EOS.undent
+      To replace ssh-agent with gpg-agent, you need to:
+         * ensure that "enable-ssh-support" is in ~/.gnupg/gpg-agent.conf
+         * ensure that "pinentry" in ~/.gnupg/gpg-agent.conf has a graphical PIN entry
+           such as pinentry-mac (needed for GUI programs)
+         * disable the system ssh-agent with
+             launchctl unload -w /System/Library/LaunchAgents/org.openssh.ssh-agent.plist
+         * have launchd start gpg-agent at login
+    EOS
+  end
```

Comment:
The ssh-agent plist is on by default.  Since there is no ordering in launchd, if both are enabled it's possible for the system's org.openssh.ssh-agent.plist to execute after the gpg-agent plist, which stomps on the SSH_AUTH_SOCK environmental variable.  Has to be disabled , but only if you are using the ssh agent emulation of gpg-agent.

How do you feel about:
"If you plan to use the ssh-support of gpg-agent, you must use a graphical pinentry program (such as pinentry-mac) and disable the default system ssh-agent (launchctl unload -w /System/Library/LaunchAgents/org.openssh.ssh-agent.plist)"

(as a note, if you don't use a graphical pinentry program, and you use a program like CyberDuck, it will fail in mysterious ways because the system has no way to prompt you for your pin, which is why I felt it should be noted).